info/guide/guide-chapter-6.md

```diff
@@ -428,6 +428,22 @@ fun SomeClass.deleteAllSpaces() {
 }
 ```
 
+#### <a name="r6.2.4"></a> 6.2.4 Don't use property length with operation - 1
```

Comment:
"Use `lastIndex` in case you need to get latest element of a collection"

Reply:
Done